app/workers/hca/submission_failure_email_analytics_job.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module HCA
+  class SubmissionFailureEmailAnalyticsJob
+    include Sidekiq::Worker
+
+    EMAIL_SUBJECT = "We didn't receive your application"
+
+    def initialize
+      unless FeatureFlipper.send_email?
+        raise Common::Exceptions::ParameterMissing.new(
+          'GovDelivery token or server',
+          detail: 'It should be configured in settings.yml'
+        )
+      end
+      if Settings.google_analytics.tracking_id.blank?
+        raise Common::Exceptions::ParameterMissing.new(
+          'Google Analytics tracking ID',
+          detail: 'It should be configured in settings.yml'
+        )
+      end
+      @tracker = Staccato.tracker(Settings.google_analytics.tracking_id)
+      @time_range_start = 1445.minutes.ago
+      @time_range_end = 5.minutes.ago
```

Comment:
Looking at emails from the past 24 hours.  Giving a 5 minute buffer so that we reduce the chance we don't get back emails that haven't actually finished processing.